react/components/LanguagePicker.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Dispatch, FC, SetStateAction } from 'react'
+import { InjectedIntl } from 'react-intl'
+
+import { Dropdown } from 'vtex.styleguide'
+
+function getLanguageOptions(intl: InjectedIntl) { 
+  return [
+    {
+      label: intl.formatMessage({
+        id: 'admin/messages.messages-upload.pick-lang.en-us',
+      }),
+      value: 'en-US',
```

Comment:
Is it better to pass `en-US` or just `en`? What is the difference?

Reply:
In this case `en` is better, since there are no other `en-*` options.